src/main/java/org/springframework/social/bitbucket/api/impl/BitBucketTemplate.java

```diff
@@ -29,6 +29,8 @@
 
     private UsersOperations usersOperations;
 
+    private GroupsOperations groupsOperations;
```

Comment:
I tend to dislike final classes unless there is a very good reason to ensure it will never be extended. Disregard the comment about final fields, it does not add much and would require inlining the call to `initSubApis()`. Probably not worth it.